app/server.R

```diff
@@ -0,0 +1,138 @@
+# Define the server code
+server <- function(input, output, session) {
+
+  disclaimer_message <- modalDialog(
+    title = "Disclaimer",
+    disclaimerpopupHTML()
+    )
+  
+  # Show the model on start up ...
+  showModal(disclaimer_message)
+  
+  # function to make request to API
+  hit_api<- eventReactive(input$go, {
```

Comment:
After the first hit of the API, how do you feel about making it Reactive not eventReactive so that it can respond to user actions like tick/untick, sliding up/down the number of close contacts?

Reply:
I think that could work. I will have to do some googling about how to implement the functionality of _After the first hit of the API making it Reactive_